myJoin.pl

```diff
@@ -24,7 +24,7 @@
 my @headers;
 my @uniq;
 my @dup;
-my @field;
+my @fields;
```

Comment:
`@field` was changed to `@fields`, because there already exists a `$field` variable. This isn't syntactically incorrect, but it is an error-prone way of coding--it is easier to keep track of variables if no array/hash/scalar shares the same name.